src/backend/src/services/projects.services.ts

```diff
@@ -15,8 +15,30 @@ import {
 } from '../utils/projects.utils';
 import { descBulletConverter, wbsNumOf } from '../utils/utils';
 import { createDescriptionBulletChangesJson } from '../utils/work-packages.utils';
+import WorkPackagesService from './work-packages.services';
 
 export default class ProjectsService {
+  /**
+   * Delete the the project in the database along with all its work packages.
+   * @returns the project that is deleted.
+   */
+  static async deleteProject(user: User, _wbsNumber: WbsNumber): Promise<WbsNumber> {
```

Comment:
same thing about being last in the file

also `_wbsNumber` should have no underscore -- underscore is for a parameter that doesn't get used